src/main/java/com/royalrangers/enums/Messages.java

```diff
@@ -0,0 +1,7 @@
+package com.royalrangers.enums;
+
+public enum Messages {
+     NOT_CONFIRMED,
```

Comment:
You can write like this NOT_CONFIRMED("some message") and don't create additional method for getting message into your service. Read about enums here: https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html